symphony-bdk-http/symphony-bdk-http-jersey2/src/main/java/com/symphony/bdk/http/jersey2/ApiClientBuilderJersey2.java

```diff
@@ -176,6 +200,12 @@ private ClientConfig createClientConfig() {
     clientConfig.property(HttpUrlConnectorProvider.SET_METHOD_WORKAROUND, true);
     // turn off compliance validation to be able to send payloads with DELETE calls
     clientConfig.property(ClientProperties.SUPPRESS_HTTP_COMPLIANCE_VALIDATION, true);
+    // By default PoolingHttpClientConnectionManager, if not configured, has 20 connection in the
+    // pool BUT only 2 max connection per route.
+    PoolingHttpClientConnectionManager connectionManager = new PoolingHttpClientConnectionManager();
```

Comment:
Any chance we could use the PoolingHttpClientConnectionManager with the TTL params and a config to allow setting? This may also help by allowing how long a connection can live.
https://hc.apache.org/httpcomponents-client-ga/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html#PoolingHttpClientConnectionManager(long,%20java.util.concurrent.TimeUnit)

Reply:
The PR is already merged and we are releasing a new version. Maybe we can discuss later if you have still problems...

Reply:
keep-alive configuration is also a feature we EIS teams needs. looking forward to having such feature soon

Reply:
Could you please create a new [Github issue](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues/new/choose)?

Reply:
@symphony-thibault ,  thx,  and here it is https://perzoinc.atlassian.net/browse/PLAT-10523